.github/workflows/runtime-extrinsics.yml

```diff
@@ -0,0 +1,69 @@
+name: Check Runtime and Extrinsics
```

Comment:
Test: remove an extrinsic from https://github.com/gluwa/creditcoin/pull/216. 

Both checks reported PASS while we expect them to report FAIL !

Reply:
Retested after committing suggestion from @nathanwhit. Both checks still report PASS instead of FAIL when extrinsics are removed, see https://github.com/gluwa/creditcoin/pull/216

Reply:
It returned with `exit 1` but passed because of the condition. I have removed the `continue_on_error` condition that was on that particular step.

Reply:
A new port was opened and both nodes started listening.
The exit code was caused by the detected extrinsic changes for verifyTransfer:
```
[!] Extrinsics indexing/ordering has changed in the creditcoin-node runtime! If this change is intentional, please bump transaction_version in lib.rs. Changed extrinsics:
[verifyTransfer] idx: 13 -> 12 (args: 1)
```

Reply:
I see this now thanks. It's been mixed up with the rest of the logs.